www/js/lib/zimfile.js

```diff
@@ -21,6 +21,20 @@
  */
 'use strict';
 
+/**
+ * Add Polyfill currently required by IE11 to run zstddec-asm and xzdec-asm
+ * See https://github.com/emscripten-core/emscripten/issues/14700
+ * If this is resolved upstream, remove this polyfill
```

Comment:
It would be good to say where this code has been copied from (as it's not ours). I know it's only a few lines, but it's always better

Reply:
Source (MDN) added.